Use the glib CFLAGS and LDFLAGS during build where necessary
https://github.com/DinoTools/dionaea/issues/161
https://github.com/DinoTools/dionaea/pull/160
Merged in in https://github.com/DinoTools/dionaea/commit/1748f3b3936aa1da2d92500251ae8010fe181dfc
diff -ru dionaea-793accd84432a77309fa8b81e1f5e9b5bd9ee7a3.orig/modules/curl/Makefile.am dionaea-793accd84432a77309fa8b81e1f5e9b5bd9ee7a3.new/modules/curl/Makefile.am
--- dionaea-793accd84432a77309fa8b81e1f5e9b5bd9ee7a3.orig/modules/curl/Makefile.am	2016-11-14 23:26:58.000000000 +0100
+++ dionaea-793accd84432a77309fa8b81e1f5e9b5bd9ee7a3.new/modules/curl/Makefile.am	2018-03-21 10:34:08.529280378 +0100
@@ -5,9 +5,12 @@
 AM_CFLAGS = -I$(top_srcdir)/include -I$(top_srcdir)/src
 AM_CFLAGS += -fno-strict-aliasing
 
-AM_CFLAGS += $(LIB_CURL_CFLAGS) 
+AM_CFLAGS += $(LIB_CURL_CFLAGS)
 AM_LDFLAGS = $(LIB_CURL_LIBS)
 
+AM_CFLAGS += $(LIB_GLIB_CFLAGS)
+AM_LDFLAGS += $(LIB_GLIB_LDFLAGS)
+
 pkglib_LTLIBRARIES = curl.la
 
 curl_la_SOURCES = module.c module.h
diff -ru dionaea-793accd84432a77309fa8b81e1f5e9b5bd9ee7a3.orig/modules/emu/Makefile.am dionaea-793accd84432a77309fa8b81e1f5e9b5bd9ee7a3.new/modules/emu/Makefile.am
--- dionaea-793accd84432a77309fa8b81e1f5e9b5bd9ee7a3.orig/modules/emu/Makefile.am	2016-11-14 23:26:58.000000000 +0100
+++ dionaea-793accd84432a77309fa8b81e1f5e9b5bd9ee7a3.new/modules/emu/Makefile.am	2018-03-21 10:34:08.529280378 +0100
@@ -8,9 +8,13 @@
 AM_CFLAGS += $(LIB_EMU_CFLAGS)
 AM_LDFLAGS = $(LIB_EMU_LIBS)
 
+AM_CFLAGS += $(LIB_GLIB_CFLAGS)
+AM_LDFLAGS += $(LIB_GLIB_LDFLAGS)
+
+
 
 pkglib_LTLIBRARIES = emu.la
 
 emu_la_SOURCES = module.c module.h detect.c emulate.c profile.c hooks.c
 
-emu_la_LDFLAGS = -module -no-undefined -avoid-version ${AM_LDFLAGS} 
+emu_la_LDFLAGS = -module -no-undefined -avoid-version ${AM_LDFLAGS}
diff -ru dionaea-793accd84432a77309fa8b81e1f5e9b5bd9ee7a3.orig/modules/nc/Makefile.am dionaea-793accd84432a77309fa8b81e1f5e9b5bd9ee7a3.new/modules/nc/Makefile.am
--- dionaea-793accd84432a77309fa8b81e1f5e9b5bd9ee7a3.orig/modules/nc/Makefile.am	2016-11-14 23:26:58.000000000 +0100
+++ dionaea-793accd84432a77309fa8b81e1f5e9b5bd9ee7a3.new/modules/nc/Makefile.am	2018-03-21 10:34:39.534850615 +0100
@@ -7,6 +7,10 @@
 AM_CFLAGS += 
 AM_LDFLAGS = 
 
+AM_CFLAGS += $(LIB_GLIB_CFLAGS)
+AM_LDFLAGS += $(LIB_GLIB_LDFLAGS)
+
+
 
 pkglib_LTLIBRARIES = nc.la
 
diff -ru dionaea-793accd84432a77309fa8b81e1f5e9b5bd9ee7a3.orig/modules/nfq/Makefile.am dionaea-793accd84432a77309fa8b81e1f5e9b5bd9ee7a3.new/modules/nfq/Makefile.am
--- dionaea-793accd84432a77309fa8b81e1f5e9b5bd9ee7a3.orig/modules/nfq/Makefile.am	2016-11-14 23:26:58.000000000 +0100
+++ dionaea-793accd84432a77309fa8b81e1f5e9b5bd9ee7a3.new/modules/nfq/Makefile.am	2018-03-21 10:34:08.530280363 +0100
@@ -5,8 +5,8 @@
 AM_CFLAGS = -I$(top_srcdir)/include -I$(top_srcdir)/src -pipe -D _GNU_SOURCE -fno-strict-aliasing
 AM_CFLAGS += -Wall -Werror
 
-AM_CFLAGS += ${LIB_GLIB_CFLAGS} 
-AM_LDFLAGS = ${LIB_GLIB_LIBS} 
+AM_CFLAGS += ${LIB_GLIB_CFLAGS}
+AM_LDFLAGS = ${LIB_GLIB_LIBS}
 
 
 pkglib_LTLIBRARIES = nfq.la
diff -ru dionaea-793accd84432a77309fa8b81e1f5e9b5bd9ee7a3.orig/modules/nl/Makefile.am dionaea-793accd84432a77309fa8b81e1f5e9b5bd9ee7a3.new/modules/nl/Makefile.am
--- dionaea-793accd84432a77309fa8b81e1f5e9b5bd9ee7a3.orig/modules/nl/Makefile.am	2016-11-14 23:26:58.000000000 +0100
+++ dionaea-793accd84432a77309fa8b81e1f5e9b5bd9ee7a3.new/modules/nl/Makefile.am	2018-03-21 10:34:08.530280363 +0100
@@ -10,6 +10,8 @@
 #AM_LDFLAGS = -Wl,-rpath,/opt/dionaea/lib -L/opt/dionaea/lib -lnl -lnl-route -lnl-genl -lnl-nf
 AM_LDFLAGS = $(LIB_NL_LIBS)
 
+AM_CFLAGS += $(LIB_GLIB_CFLAGS)
+AM_LDFLAGS += $(LIB_GLIB_LDFLAGS)
 
 pkglib_LTLIBRARIES = nl.la
 
diff -ru dionaea-793accd84432a77309fa8b81e1f5e9b5bd9ee7a3.orig/modules/pcap/Makefile.am dionaea-793accd84432a77309fa8b81e1f5e9b5bd9ee7a3.new/modules/pcap/Makefile.am
--- dionaea-793accd84432a77309fa8b81e1f5e9b5bd9ee7a3.orig/modules/pcap/Makefile.am	2016-11-14 23:26:58.000000000 +0100
+++ dionaea-793accd84432a77309fa8b81e1f5e9b5bd9ee7a3.new/modules/pcap/Makefile.am	2018-03-21 10:34:08.531280350 +0100
@@ -8,6 +8,9 @@
 AM_CFLAGS += $(LIB_PCAP_CFLAGS)
 AM_LDFLAGS = $(LIB_PCAP_LIBS)
 
+AM_CFLAGS += $(LIB_GLIB_CFLAGS)
+AM_LDFLAGS += $(LIB_GLIB_LDFLAGS)
+
 pkglib_LTLIBRARIES = pcap.la
 
 pcap_la_SOURCES = pcap.c
diff -ru dionaea-793accd84432a77309fa8b81e1f5e9b5bd9ee7a3.orig/modules/xmatch/Makefile.am dionaea-793accd84432a77309fa8b81e1f5e9b5bd9ee7a3.new/modules/xmatch/Makefile.am
--- dionaea-793accd84432a77309fa8b81e1f5e9b5bd9ee7a3.orig/modules/xmatch/Makefile.am	2016-11-14 23:26:58.000000000 +0100
+++ dionaea-793accd84432a77309fa8b81e1f5e9b5bd9ee7a3.new/modules/xmatch/Makefile.am	2018-03-21 10:34:08.531280350 +0100
@@ -8,6 +8,9 @@
 AM_CFLAGS += $(LIB_XMATCH_CFLAGS)
 AM_LDFLAGS = $(LIB_XMATCH_LIBS)
 
+AM_CFLAGS += $(LIB_GLIB_CFLAGS)
+AM_LDFLAGS += $(LIB_GLIB_LDFLAGS)
+
 pkglib_LTLIBRARIES = xmatch.la
 
 xmatch_la_SOURCES = module.c xmatch.c
